File: app/src/main/java/com/ponthaitay/listmovie/kotlin/SimpleActivity.java
package com.ponthaitay.listmovie.kotlin;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class SimpleActivity extends AppCompatActivity {

    private TextView tvText;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_simple);
        tvText = (TextView) findViewById(R.id.tv_text);
        title("20Scoops CNX", tvText);
    }

    private void title(String title, TextView textView) {
        textView.setText(title);
    }
}

File: app/src/main/java/com/ponthaitay/listmovie/kotlin/kotlin/SimpleActivity.kt
package com.ponthaitay.listmovie.kotlin.kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView
import com.ponthaitay.listmovie.kotlin.R
import kotlinx.android.synthetic.main.activity_simple.*

infix fun TextView.tile(title: String?) = {
    this.text = title
}

class SimpleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_simple)
        tv_text tile ("20Scoops CNX")
    }
}


